src/main/java/me/eccentric_nz/TARDIS/database/TARDISSQLiteDatabase.java

```diff
@@ -231,6 +231,10 @@ public void createTables() {
             String queryVortex = "CREATE TABLE IF NOT EXISTS " + prefix + "vortex (tardis_id INTEGER PRIMARY KEY NOT NULL, task INTEGER DEFAULT 0)";
             statement.executeUpdate(queryVortex);
 
+            // Table structure for table 'traveled_to'
```

Comment:
All the rest of the CREATE statements are in alphabetical order... just makes it easier to find if I make changes later